next/pages/form/[eform].tsx

```diff
@@ -94,15 +93,59 @@ const FormTestPage = ({
       })
     })
   }
+  const validateDateFromToFormat = (formData: any, errors: FormValidation) => {
+    const formDataKeys = Object.keys(formData)
+    formDataKeys?.forEach((key) => {
+      if (
+        form?.currentSchema?.properties[key]?.dateFromTo &&
```

Comment:
this assumes this data lives only on 'top level' of each step, which might not be true. But same issue is in the other validation, I'm ok with fixing this all at once and keeping like this for now, just pointing this out.